src/iframe/src/canvasAPI/print.js

```diff
@@ -1,70 +1,26 @@
-import colors from '../colors.js'
 import alphabet from './alphabet.js'
 
-const print = ({ x, y, letters, c, ctx }) => {
-  const color = colors.triplet(c)
-
-  const grids = letters
-    .toString()
-    // Split into individual letters.
-    .split('')
-    // Get the pixels and the letter's width.
-    .map(letter => {
-      const pixels = alphabet[letter.toLowerCase()]
-      return {
-        // If we found a matching letter in the alphabet,
-        // return its width.
-        // Otherwise if the user tries to print a character
-        // we haven't defined, just return 3.
-        width: pixels ? pixels.length / 6 : 3,
-        letter,
-        pixels
-      }
-    })
-    // Calculate running offsets.
-    .reduce((acc, current, index) => {
-      const previous = acc[index - 1]
-      const xOffset = previous ? previous.width + 1 + previous.xOffset : 0
-      const yOffset = previous ? previous.yOffset : 0
-
-      return [
-        ...acc,
-        {
-          ...current,
-          xOffset,
-          yOffset
+const drawText = ({ x, y, letters, c, setPixel, cameraX, cameraY }) => {
+  let currentX = Math.floor(x - cameraX)
+  let currentY = Math.floor(y - cameraY)
+
+  for (let letter of letters.toString().split('')) {
```

Comment:
In general, would you be able to add comments wherever possible? I try to do it every line, or close to every line. I don't trust I will remember what I did more than 2 weeks ago. ;)

Reply:
Of course. I will comment this stuff a bunch more tonight.